mod/wikirate/set/right/contributed_analysis.rb

```diff
@@ -1,6 +1,6 @@
 def contribution_count
   @cc ||= Card.search :type_id=>WikirateAnalysisID,
-            :right_plus=>[ Card[:wikirate_article].name, {:edited_by=>cardname.left }],
+            :right_plus=>[ Card[:overview].name, {:edited_by=>cardname.left }],
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside square brackets detected.<br>Space inside { missing.